state/futures/types.ts

```diff
@@ -153,9 +160,7 @@ export type CrossMarginState = {
 	positions: {
 		[account: string]: FuturesPosition<string>[];
 	};
-	openOrders: {
-		[account: string]: FuturesOrder<string>[];
-	};
+	openOrders: DelayedOrder<string>[];
```

Comment:
Would prefer if we kept it separated by account, this way we can have much smoother switching when changing wallets